nvm.sh

```diff
@@ -86,8 +86,12 @@ nvm_has_colors() {
   [ "${NVM_NUM_COLORS:--1}" -ge 8 ]
 }
 
+nvm_curl() {
+  command curl -q "$@"
+}
+
 nvm_curl_libz_support() {
-  curl -V 2>/dev/null | nvm_grep "^Features:" | nvm_grep -q "libz"
+  nvm_curl -V 2>/dev/null | nvm_grep "^Features:" | nvm_grep -q "libz"
```

Comment:
this invocation doesn't need `-q`, so this should be sufficient:
```suggestion
  command curl -V 2>/dev/null | nvm_grep "^Features:" | nvm_grep -q "libz"
```